### DISCUSSION TITLE
Disable auto-logout for administrators

### DISCUSSION THREAD

**Discussion by francwalter** [kimai/kimai#3707] [Questions & Answers]
Hallo
as I read in #1026 admins will logout for security reasons after a while and cannot be remembered, as in [documentation > remeber-me-login](https://www.kimai.org/documentation/users.html#remember-me-login).
Is there a way to disable this feature? 
I use kimai2 as super-admin and only on my local computer and never in public, so it is easier to stay logged in without any logout.

In my php.ini I have:
`session.gc_maxlifetime = 1440`
This is 24 Minutes only. 
I dont know if it is good to set e.g. 2592000 (for 30 days) here, maybe not so good for the server performance, I dont know.

Thank for hints.
frank

**Comment by kevinpapst** [comment]
Session timeout is not a Kimai setting. Do as you wish. It has NOTHING to do with performance.
Next version will have a proper remember me implementation, the current one is partially broken.
